src/data/storage/StorageDataStoreClient.ts

```diff
@@ -72,4 +72,49 @@ export class StorageDataStoreClient extends StorageClient {
     public async listKeys(): Promise<string[]> {
         return this.dataStore.getKeys().getData();
     }
+
+    public async getObjectSharing(key: string): Promise<ObjectSharing | undefined> {
+        const metadata = await this.getMetadataByKey(key);
+
+        const sharing = _.omit(
+            metadata,
+            "created",
+            "lastUpdated",
+            "lastUpdatedBy",
+            "namespace",
+            "key",
+            "value",
+            "favorite",
+            "id"
+        );
+
+        return { ...sharing, user: { ...sharing.user, name: "" } };
+    }
+
+    public async saveObjectSharing(key: string, object: ObjectSharing): Promise<void> {
+        const metadata = await this.getMetadataByKey(key);
+
+        await this.api.post(`/sharing?type=dataStore&id=${metadata.id}`, undefined, {
+            object,
+        }).response;
+    }
+
+    private async getMetadataByKey(key: string) {
+        const response = await this.api.get<MetadataDataStoreKey>(
+            `/dataStore/${dataStoreNamespace}/${key}/metaData`
+        ).response;
+
+        return response.data;
```

Comment:
Instead of using ``response.data`` we have the helper method ``getData`` on the API:

```ts
        const data = await this.api
            .get<DataStoreMetadata>(`/dataStore/${dataStoreNamespace}/${key}/metaData`)
            .getData();

        return data;
```

Reply:
Ok, fixed!